waisntechtools/alerts/urls.py

```diff
@@ -10,4 +10,6 @@
     path('alert/sent', alert_request.sent, name='alert_sent'),
     path('follow-up', follow_up_request.index, name='follow_up'),
     path('follow-up/sent', follow_up_request.sent, name='follow_up_sent'),
+    path('twilio-endpoint',
```

Comment:
Could you keep the same end-point that was there before since the project has been deployed? This is a better name, which I think is worth changing, but something that I think makes sense to do by itself so that we can have a focused roll out of this change since it'll require configuration change in Twilio to happen simultaneously during deployment.